src/controllers/albums.js

```diff
@@ -0,0 +1,98 @@
+const db = require('../db/index')
+
+const createAlbum = async (req, res) => {
+    try {
+        const { name, year } = req.body
+        const artistID = req.params.id
+        const { rows: artist } = await db.query(`
+        SELECT * FROM Artists WHERE id=$1`, [artistID])
```

Comment:
You don't need backticks here as you aren't using any variables inside. Replace with single quotes. This is the same throughout the artist and album controller. If you format the file it should fix this and the indentations at the same time
```
        const { rows: artist } = await db.query('
        SELECT * FROM Artists WHERE id=$1', [artistID])
```